actors/common/src/main/java/org/sunbird/learner/actors/InitialisationActor.java

```diff
@@ -0,0 +1,265 @@
+    Util.initializeContext(request, JsonKey.ORGANISATION);
+    // set request id fto thread loacl...
+    ExecutionContext.setRequestId(request.getRequestId());
+    if (request.getOperation().equalsIgnoreCase(ActorOperations.CREATE_FIRST_ROOTORG.getValue())) {
+      createFirstRootOrg(request);
+    } else {
+      onReceiveUnsupportedOperation(request.getOperation());
+    }
+  }
+  /** Method to create an first root organization. */
+  @SuppressWarnings("unchecked")
+  private void createFirstRootOrg(Request actorMessage) {
+    ProjectLogger.log("createFirstRootOrg method call started");
+    checkIfAlreadyInitialised();   
+    try {
+      Map<String, Object> req =
+          (Map<String, Object>) actorMessage.getRequest().get(JsonKey.ORGANISATION);
+     
+      Util.DbInfo orgDbInfo = Util.dbInfoMap.get(JsonKey.ORG_DB);
+	  
+      if (req.containsKey(JsonKey.CHANNEL)) {
+        validateChannel(req);
+      }    
+
+      String updatedBy = "INITIALIZER";
+      req.put(JsonKey.CREATED_BY, updatedBy);
+      String uniqueId = ProjectUtil.getUniqueIdFromTimestamp(actorMessage.getEnv());
+      req.put(JsonKey.ID, uniqueId);
+      req.put(JsonKey.ROOT_ORG_ID, uniqueId);
+      req.put(JsonKey.HASHTAGID, uniqueId);
+      req.put(JsonKey.CREATED_DATE, ProjectUtil.getFormattedDate());
+      req.put(JsonKey.STATUS, ProjectUtil.OrgStatus.ACTIVE.getValue());
+      req.put(JsonKey.IS_ROOT_ORG, true);
+      req.put(JsonKey.IS_DEFAULT,true);
+ 
+      String slug = Slug.makeSlug((String) req.getOrDefault(JsonKey.CHANNEL, ""), true);        
+      boolean bool = isSlugUnique(slug);
+          if (bool) {
+            req.put(JsonKey.SLUG, slug);
+          } else {
+            ProjectCommonException exception =
+                new ProjectCommonException(
+                    ResponseCode.slugIsNotUnique.getErrorCode(),
+                    ResponseCode.slugIsNotUnique.getErrorMessage(),
+                    ResponseCode.CLIENT_ERROR.getResponseCode());
+            sender().tell(exception, self());
+            return;
+          }     
+    
+        boolean channelRegBool = Util.registerChannel(req);
+        if (!channelRegBool) {
+          ProjectCommonException exception =
+              new ProjectCommonException(
+                  ResponseCode.channelRegFailed.getErrorCode(),
+                  ResponseCode.channelRegFailed.getErrorMessage(),
+                  ResponseCode.SERVER_ERROR.getResponseCode());
+          sender().tell(exception, self());
+          return;
+        }
+      
+      // This will remove all extra unnecessary parameter from request
+      Organization org = mapper.convertValue(req, Organization.class);
+      req = mapper.convertValue(org, Map.class);
+      Response result =
+          cassandraOperation.insertRecord(orgDbInfo.getKeySpace(), orgDbInfo.getTableName(), req);
+      // save initialisation status to system settings file    
+      addInitialisationFlagToSystemSettings();
```

Comment:
Create Dao and Service classes for system_settings table.

SystemSetting.java (model)

SystemSettingDao.java
SystemSettingDaoImpl.java

SystemSettingService.java
SystemSettingServiceImpl.java

SystemInitActor will use SystemSettingService to perform following operations.

saveSetting
getSetting